Source/monster.h

```diff
@@ -25,8 +25,8 @@ namespace devilution {
 
 struct Missile;
 
-#define MAXMONSTERS 200
-#define MAX_LVLMTYPES 24
+constinit const int MaxMonsters = 200;

```

Comment:
`constexpr`, because not all of our targets support `constinit`. Also, no need for `const`, it is implied by `constinit`/`constexpr`.

Reply:
This isn't actually part of this PR and only there because @aetn23 merged rather then create a new branch/rebase after his previous PR got merged.

Reply:
Yes, this was resolved a commit or two later and is already merged to master.